core/src/main/java/bio/terra/pearl/core/model/kit/KitOriginType.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.pearl.core.model.kit;
+
+public enum KitOriginType {
+    SHIPPED, //kit was shipped to the participant by GP
```

Comment:
Wondering if "SHIPPED" should just be "DSM" instead to provide more specificity in the future when we may have other kit providers integrated

Reply:
"MAILED"?

Reply:
It feels like "MAILED"/"IN_PERSON" is separate from, perhaps, `KitProvider` which could be `GP` (since we're moving away from DSM) `External` etc.